### PR TITLE
Feature/upload img companies

### PR DESCRIPTION
**Pull request by ganout**

### CODE REVIEW

File: src/main/java/com/wildcodeschool/ressource/storage/FileSystemStorageService.java [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+
+    /**
+     * Get a stream of all the paths in the storage.location directory
+     * (except the storage.location itself)
+     *
+     * @return a Stream<Path> the paths
+     * @throws StorageException if the storage.location can't be red
+     */
+    /*@Override
+    public Stream<Path> loadAll() {
```
Comment (bastienwcs) [inline src/main/java/com/wildcodeschool/ressource/storage/FileSystemStorageService.java:64]: Code mort :(

File: src/main/java/com/wildcodeschool/ressource/storage/StorageProperties.java [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.boot.context.properties.ConfigurationProperties;
+
+@ConfigurationProperties("storage")
+public class StorageProperties {
+
+    /**
+     * Folder location for storing files
+     */
+    /*@Value("${upload.path.company}")
```
Comment (bastienwcs) [inline src/main/java/com/wildcodeschool/ressource/storage/StorageProperties.java:16]: Code mort

File: src/main/java/com/wildcodeschool/ressource/storage/FileSystemStorageService.java (hunk 1 above)
Comment (ganout) [reply src/main/java/com/wildcodeschool/ressource/storage/FileSystemStorageService.java]: it's ok

File: src/main/java/com/wildcodeschool/ressource/storage/StorageProperties.java (hunk 2 above)
Comment (ganout) [reply src/main/java/com/wildcodeschool/ressource/storage/StorageProperties.java]: it's ok

**Pull request merged**